<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DockWidget</class>
 <widget class="QDockWidget" name="DockWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>750</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1200</width>
    <height>750</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1200</width>
    <height>750</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widget1">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>50</y>
      <width>201</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 8pt &quot;MS Shell Dlg 2&quot;;
font: 75 30pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Welcome</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushbtnClear">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>580</y>
      <width>201</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;MS Shell Dlg 2&quot;;
Background-color: rgb(255, 255, 255);
border-radius: 15px;</string>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="txtEditName">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>220</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">Background-color: rgb(255, 255, 255);
border-radius: 10px;</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="txtEditAge">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>360</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">Background-color: rgb(255, 255, 255);
border-radius: 10px;</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="txtEditMobile">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>220</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">Background-color: rgb(255, 255, 255);
border-radius: 10px;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>220</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 16pt &quot;MS Shell Dlg 2&quot; ;</string>
    </property>
    <property name="text">
     <string>Name</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>360</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 16pt &quot;MS Shell Dlg 2&quot; ;</string>
    </property>
    <property name="text">
     <string>Gender</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>220</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 16pt &quot;MS Shell Dlg 2&quot; ;</string>
    </property>
    <property name="text">
     <string>Mobile</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>360</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 16pt &quot;MS Shell Dlg 2&quot; ;</string>
    </property>
    <property name="text">
     <string>Age</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radbtnMale">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>360</y>
      <width>95</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 16pt &quot;MS Shell Dlg 2&quot; ;</string>
    </property>
    <property name="text">
     <string>Male</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radbtnFemale">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>360</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 16pt &quot;MS Shell Dlg 2&quot; ;</string>
    </property>
    <property name="text">
     <string>Female</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushbtnRegister">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>580</y>
      <width>201</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;MS Shell Dlg 2&quot;;
Background-color: rgb(255, 255, 255);
border-radius: 15px;</string>
    </property>
    <property name="text">
     <string>Register</string>
    </property>
   </widget>
   <widget class="QLabel" name="background">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1200</width>
      <height>750</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <zorder>background</zorder>
   <zorder>label</zorder>
   <zorder>pushbtnClear</zorder>
   <zorder>txtEditAge</zorder>
   <zorder>txtEditMobile</zorder>
   <zorder>label_2</zorder>
   <zorder>label_3</zorder>
   <zorder>label_4</zorder>
   <zorder>label_5</zorder>
   <zorder>radbtnMale</zorder>
   <zorder>radbtnFemale</zorder>
   <zorder>pushbtnRegister</zorder>
   <zorder>txtEditName</zorder>
  </widget>
 </widget>
 <resources>
  <include location="test.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushbtnClear</sender>
   <signal>clicked()</signal>
   <receiver>txtEditMobile</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>753</x>
     <y>602</y>
    </hint>
    <hint type="destinationlabel">
     <x>844</x>
     <y>246</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushbtnClear</sender>
   <signal>clicked()</signal>
   <receiver>txtEditAge</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>813</x>
     <y>605</y>
    </hint>
    <hint type="destinationlabel">
     <x>474</x>
     <y>388</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushbtnClear</sender>
   <signal>clicked()</signal>
   <receiver>txtEditName</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>719</x>
     <y>595</y>
    </hint>
    <hint type="destinationlabel">
     <x>490</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
